main.nf

```diff
@@ -575,6 +577,7 @@ if(params.aligner == 'star'){
         script:
         prefix = reads[0].toString() - ~/(_R1)?(_trimmed)?(_val_1)?(\.fq)?(\.fastq)?(\.gz)?$/
         def avail_mem = task.memory == null ? '' : "--limitBAMsortRAM ${task.memory.toBytes() - 100000000}"
+        def seqCenter = params.seqCenter ? "--outSAMattrRGline ID:$prefix 'CN:$params.seqCenter'" : ''
```

Comment:
This is now triggering a syntax error in the [Travis CI tests](https://travis-ci.org/nf-core/rnaseq/jobs/415394188):

```
ERROR ~ Variable `prefix` already defined in the process scope @ line 580, column 68.
   ter ? "--outSAMattrRGline ID:$prefix 'CN
                                 ^
```